install.sh

```diff
@@ -2,6 +2,25 @@
 
 set -e
 
-echo "This installation method has been deprecated."
-echo "Please run 'pip install algorithmia' instead."
-echo "See https://algorithmia.com/developers/clients/cli for more details."
+migrate_config() {
+    if [ -f ~/.algorithmia ]; then
+        echo_verbose "migrating configuration..."
+        mv ~/.algorithmia ~/.algorithmia.bak
+        mkdir ~/.algorithmia
+        mv ~/.algorithmia.bak ~/.algorithmia/config
+    fi
+}
+
+pyv=$(python -V 2>&1 | grep -Po '(?<=Python )(.+)')
+if [[ -n "$pyv" ]]; then
+    pipv=$(pip --version | grep -Po '(?<=pip )(.+)')
```

Comment:
I don't object to this approach, but given that the version of pip doesn't matter here, I find it more elegant to just check if the command exists as it seems less cryptic than the regex bits. e.g.

```
    if ! command -v "$1" > /dev/null 2>&1; then
        echo "...pip not found..."
    fi
 ```